docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
+
+Image tag must match the regular expression `[\w][\w.-]{0,127}` where `\w` 
```

Comment:
It is probably okay to shorten this to have the expanded character class, as `\w` hides the combination.

Double check, but I think `[A-Za-z0-9_][A-Za-z0-9_-.]` is correct.